addons/website/static/src/builder/plugins/image/image_tool_option.xml

```diff
@@ -19,7 +19,7 @@
     <BuilderRow label.translate="Transform" preview="false">
         <BuilderButton title.translate="Crop image" action="'cropImage'" icon="'fa-crop'" id="'cropImage'" />
         <BuilderButton title.translate="Reset crop" action="'resetCrop'" t-if="this.isActiveItem('cropImage')">Reset</BuilderButton>
-        <ImageTransformOption />
+        <ImageTransformOption t-if="!state.isImageAnimated" />
```

Comment:
Maybe you can use something like `isActiveItem('no_animation_opt')` rather than a change on the state here so that there is no duplicate of the condition that checks if there is an animation on the image or not :+1: